crates/db_schema/src/schema_setup/triggers.sql

```diff
@@ -0,0 +1,472 @@
+-- A trigger is associated with a table instead of a schema, so they can't be in the `r` schema. This is
```

Comment:
I think only Rust files should be under src, and everything else in assets folder.

Reply:
Agree, you could also just move this folder to the migrations folder so you don't have to change the lints / sql format checks (I don't think that hurts anything). If that doesn't work, top level is fine.

Reply:
I moved it to crates/db_schema/replaceable_schema